src/projection.d.ts

```diff
@@ -112,3 +118,20 @@ export interface ProjectionOptions extends InsetOptions {
    */
   clip?: boolean | number | "frame" | null;
 }
+
+/**
+ * A materialized projection, as returned by *plot*.projection()
+ */
+export interface Projection {
+  /** The projection’s name, if specified. */
+  name?: string;
+  /** A function that projects a point coordinates. */
+  point: (point: [number, number]) => [x: number, y: number] | undefined;
```

Comment:
Yes, but I was wondering if apply should be for geometries, not points. However we can handle both types since they're easy to detect (geometries have a `.type` property).